app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -325,6 +325,7 @@ alwaysDeny=Always deny
 alwaysAsk=Always ask
 autoHideMenuBar=Hide the menu bar by default
 disableTitleMode=Always show the URL bar
+wideURLbar=Use wide URL bar
```

Comment:
@luixxiul I think that's a good name 😄 

Other names might be too complex, like "Have URL bar fill entire width" (which sounds confusing)